### DISCUSSION TITLE
Curious question about the new CMake.

### DISCUSSION THREAD

**Discussion by airvzxf** [Syllo/nvtop#274] [General]
I have a curious question about the new CMake.

It is for personal understanding and knowledge. I checked the Dockerfile, and you don't use the `CMake` version provided by `nvidia/opengl:1.2-glvnd-runtime-ubuntu18.04`. I could speculate a little and say that you prefer control to `CMake` for both old versions and new versions, providing stability in the «nvtop» construction process. Is this correct? Or is there some other underlying reason?

[Merge master into ascend branch | 5019693 | Dockerfile | Line 14](https://github.com/Syllo/nvtop/commit/50196930e30bda6445169a0a2892f2fca6198d22#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557L14)

**Comment by Syllo** [comment]
Ubuntu 18.04 does not provide a recent enough CMake version, nothing more :wink: 
Although, what I should really do is bump it to Ubuntu 20.04 since the "Standard Support" for 18.04 ended last year.

**Reply by airvzxf** [reply]
Thank you for the information.
